paddle/fluid/memory/detail/buddy_allocator_test_data

```diff
@@ -0,0 +1,3373 @@
+16128	-1
+16128	3093
+16128	116
+16128	115
+16128	-1
+16128	3097
+16128	114
+16128	-1
+16128	3100
+16128	113
+16128	99
+16128	-1
+16128	-1
+16128	3105
+16128	3104
+16128	-1
+16128	-1
+16128	3109
+16128	3108
+16128	-1
+16128	-1
+16128	3113
+16128	-1
+16128	-1
+16128	3116
+16128	3115
+16128	-1
+16128	-1
+16128	3120
+16128	-1
+16128	-1
+16128	3123
+16128	3122
+16128	-1
+16128	3126
+16128	3119
+16128	-1
+16128	3129
+16128	3112
+16128	-1
+16128	3132
+16128	98
+16128	-1
+16128	3135
+16128	97
+16128	-1
+16128	3138
+16128	94
+16128	-1
+16128	3141
+16128	93
+16128	-1
+16128	3144
+64256	-1
+16128	92
+16128	91
+16128	90
+16128	74
+64256	-1
+16128	73
+16128	72
+16128	71
+16128	70
+32256	-1
+16128	67
+3328	-1
+64256	-1
+64256	3159
+3328	3158
+3328	-1
+64256	-1
+64256	3163
+3328	3162
+32256	-1
+1280256	-1
+32256	3166
+64256	3151
+1280256	3167
+32256	-1
+1280256	-1
+32256	3171
+64256	3146
+1280256	3172
+16128	-1
+16128	-1
+16128	3177
+32256	3156
+16128	-1
+16128	-1
+16128	3181
+16128	3180
+32256	127
+16128	3176
+16128	-1
+16128	3186
+16128	-1
+16128	-1
+16128	3189
+16128	-1
+16128	3191
+320256	-1
+16128	3188
+320256	3193
+16128	-1
+16128	3196
+16128	66
+16128	-1
+16128	3199
+16128	65
+16128	64
+32256	-1
+16128	63
+16128	-1
+16128	-1
+16128	3206
+16128	3205
+16128	-1
+16128	-1
+16128	3210
+16128	-1
+16128	-1
+16128	3213
+16128	3212
+16128	-1
+16128	3216
+16128	3209
+16128	-1
+16128	3219
+16128	49
+16128	-1
+16128	3222
+16128	48
+16128	-1
+16128	3225
+64256	-1
+16128	47
+16128	44
+16128	43
+16128	42
+32256	-1
+16128	41
+32256	3232
+32256	3203
+3328	-1
+64256	-1
+64256	3237
+3328	3236
+32256	-1
+32256	3240
+32256	-1
+1280256	-1
+32256	3242
+64256	3227
+1280256	3243
+16128	-1
+16128	-1
+16128	3248
+16128	3247
+32256	100
+16128	-1
+16128	3252
+16128	-1
+16128	3254
+320256	-1
+16128	40
+320256	3256
+320256	1320
+32256	-1
+16128	39
+32256	3260
+32256	77
+320256	-1
+320256	3264
+32256	-1
+32256	3266
+8000256	-1
+320256	14
+3328	-1
+256	-1
+256	-1
+256	3272
+1280256	-1
+3328	3270
+256	-1
+5376	-1
+5376	3277
+3328	-1
+1280256	3274
+256	-1
+256	-1
+256	3282
+1280256	-1
+3328	3279
+256	-1
+5376	-1
+5376	3287
+8000256	-1
+1280256	3284
+256	-1
+5376	-1
+5376	3292
+40192	-1
+8000256	3289
+256	-1
+5376	-1
+5376	3297
+8000256	-1
+40192	3294
+256	-1
+5376	-1
+5376	3302
+256	-1
+256	-1
+256	-1
+256	3306
+256	3305
+256	-1
+256	3309
+256	-1
+256	-1
+256	3312
+3328	-1
+1280256	-1
+3328	-1
+1280256	-1
+8000256	-1
+40192	-1
+8000256	-1
+256	3311
+256	3304
+256	3301
+256	3296
+256	3291
+256	3286
+8000256	3320
+256	3281
+256	3276
+256	3271
+256	-1
+8000256	3318
+40192	3319
+8000256	3299
+1280256	3317
+3328	3316
+1280256	3315
+3328	3314
+256	3331
+256	1112
+32256	50
+32256	32
+256	1054
+256	1025
+3328	1285
+3328	1206
+32256	13
+32256	12
+32256	11
+32256	10
+16128	38
+16128	37
+16128	36
+16128	29
+16128	24
+16128	23
+16128	18
+16128	17
+3328	9
+3328	8
+1280256	1293
+1280256	1210
+3328	3
+3328	1
+1280256	2
+1280256	0
+8000256	3268
+8000256	1085
+40192	1081
+40192	6
+8000256	5
+8000256	4
```

Comment:
Remove this file which is generated by unittest.

Reply:
this is not generated by unittest, this is test data prepared for running unittest